src/Compilers/CSharp/Portable/Binder/Semantics/Operators/OperatorFacts.cs

```diff
@@ -210,5 +210,44 @@ public static string BinaryOperatorNameFromOperatorKind(BinaryOperatorKind kind,
                     throw ExceptionUtilities.UnexpectedValue(kind & BinaryOperatorKind.OpMask);
             }
         }
+
+        public static BinaryOperatorKind SyntaxKindToBinaryOperatorKind(SyntaxKind kind)
```

Comment:
>public static BinaryOperatorKind SyntaxKindToBinaryOperatorKind(SyntaxKind kind) [](http://example.com/codeflow?start=8&length=80)

Please move helpers to their original place in their original syntactic form (no behavior changes - no code changes), simply adjust their accessibility as appropriate.